### PR TITLE
add fakeFirstName and fakeLastName

### PR DESCRIPTION
**Pull request by jalakoo**
I often need first and last names independently (to test a form, for example). So I broke up the randomFakeName() function into 3: 
- randomFirstName
- randomLastName
- updated randomFakeName

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,14 @@ func asMethods() {
 	// Example Output: "Megan Freeman"
 	Randoms.randomFakeName()
 
+    // Return random first name
+    // Example Output: "Megan"
+    Randoms.randomFakeFirstName()
+
+    // Return random last name
+    // Example Output: "Freeman"
+    Randoms.randomFakeLastName()
```
Comment (thellimist) [inline README.md]: spacing is wrong

File: SwiftRandom/Randoms.swift [hunk 2]
```diff
@@ -267,9 +267,17 @@ public struct Randoms {
     //==========================================================================================================
 
     public static func randomFakeName() -> String {
+        return randomFakeFirstName() + " " + randomFakeLastName()
```
Comment (thellimist) [inline SwiftRandom/Randoms.swift]: Could you please test this. I think it's supposed to be `Randoms.randomFakeFirstName()` etc.

File: README.md (hunk 1 above)
Comment (jalakoo) [reply README.md]: Adjusted in updated pull request.

File: SwiftRandom/Randoms.swift (hunk 2 above)
Comment (jalakoo) [reply SwiftRandom/Randoms.swift]: Passed updated unit tests and also manually checked. No compiler error. Not sure when, but at least as of Xcode 9 the compiler knows to associate static/class calls like this to the enclosing class type.

**Pull request merged**